content/kubeone/main/guides/mirror-registries/_index.en.md

```diff
@@ -115,22 +87,40 @@ registriesAuth: |
         password: "bridge"
 ```
 
-Make sure to have containerd explicitly enabled in the KubeOneCluster manifest:
-
-```yaml
-...
-containerRuntime:
-  containerd: {}
-```
-
 The credentials file is provided to the `kubeone apply` command using the
 `--credentials`/`-c` flag, such as:
 
 ```
 kubeone apply --manifest kubeone.yaml --credentials credentials.yaml
 ```
 
+## docker.io pull rate limitations
+
+docker.io registry introduced pretty low rate limits for unauthenticated requests. There are few workarounds:
+
+* Buy docker subscribtion.
+* Setup own pull-through caching proxy.
+* Use public pull-through caching proxy.
+
+### Configuring Public Pull-through Caching Proxy
```

Comment:
2 and 3 are covered with your example but it'd be also great if we can document 1 and how to setup KubeOne/KKP to use credentials to pull docker.io images

Reply:
There is a section above on how to configure credentials for a given registry. Added a link to that section.